helpers/passwordReset.js

```diff
@@ -0,0 +1,66 @@
+const nodemailer = require('nodemailer');
+const crypto = require('crypto');
+
+const config = require('../config/options');
+
+const url = 'https://www.skybunk.xyz';
+
+
+module.exports = {
+	sendPasswordResetEmail(user, email){
+		return new Promise((res, rej) => {
+			const token = crypto.randomBytes(20).toString('hex');
+
+			var mailOptions;
+			if(user.info.email == undefined || user.info.email == ''){ //send email to webmasters and tell them to verify the password reset was requested
+				mailOptions = {
+					to: config.webmasterEmail,
+					subject: `Skybunk Password Reset for ${user.firstName} ${user.lastName}`,
+					text:
+					`You are receiving this because ${user.firstName} ${user.lastName}  has requested the reset of the password for your Skybunk account. `
```

Comment:
Super pedantic spacing nitpick (should be one space before "has requested", not two?)